src/conversion.jl

```diff
@@ -20,15 +20,19 @@ GI.ncoord(::GI.PointTrait, p::Point) = embeddim(p)
 GI.getcoord(::GI.PointTrait, p::Point) = coordinates(p)
 GI.getcoord(::GI.PointTrait, p::Point, i) = coordinates(p)[i]
 
-GI.ngeom(::Any, s::Segment) = nvertices(s)
-GI.getgeom(::Any, s::Segment, i) = vertices(s)[i]
+GI.ncoord(::GI.LineTrait, s::Segment) = embeddim(s)
```

Comment:
So the GI.ncoord must be defined for geometries too? I thought it was only defined for GI.PointTrait,at least that was the concept I remember from the last time I read the GI docs.

Reply:
Yes, writing was not working because that was missing because the other packages use `GI.is3d` which depend of `GI.ncoord`. Hence, defining `GI.ncoord` helps with the conversion from `Meshes` to other packages to be able to write to other formats.